### ISSUE TITLE
Multiple auth methods

### ISSUE THREAD

**Issue by Toucan-Sam** [Fallenbagel/jellyseerr#100]
### Description

At the moment it looks like you can only use Jellyfin OR Plex, not both. Ref: #34

See: https://github.com/Fallenbagel/jellyseerr/issues/34#issuecomment-1119096593

In a perfect world both Plex and Jellyfin auth methods would work. I'm probably an edge case, but I do have both Plex and Jellyfin running because they don't both have apps for every platform (yet).

Ultimately I'd like to switch to LDAP eventually.

It's not critical and I realize some of the functionality is via overseerr, but I think it's worth leaving an issue open in case anyone wants to contribute.

### Desired Behavior

Log in with whatever platform.

### Additional Context

Long term goal, which may fall more into overseerr territory, but the requests features provided by overseerr/jellyseerr are wonderful and it'd be nice to be platform agnostic.

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Toucan-Sam** [comment]
From @Fallenbagel :
Hi. It works both on latest and develop rn since v1.1.0. But yes multi instances is still not a thing

**Comment by Fallenbagel** [comment]
Yes unfortunately it's not due to the fact that Overseerr/Jellyseerr does not work as it's own thing and is completely dependent on a single mediaserver.

There has been talks of decoupling Overseerr from this dependence and that could essentially enable us to use it simultaneously

**Comment by butts136** [comment]
Would be really nice, because some users wants Plex for sharing multiple libraries from multiple server and some want Jellyfin for their free Android and iOS app.

But because i have both installed on my server, jellyseer only allows me to connect with Plex -_-

Thanks

**Comment by xenago** [comment]
This would be awesome! I set it up and was very confused because the docs all show multiple methods for jf/plex/emby but in the UI there aren't others available once it's set up with a specific media server. I assume that's probably explained somewhere but it was not obvious to me until I set it up and found this issue haha

**Comment by zazabozaza** [comment]
Thats really the only thing missing from Jellyseerr atm imo. Cant wait for this to become a reality!

**Comment by Marcbacca** [comment]
I'd love to see this get implemented. I have Jellyfin as a Plex backup as things slowly get shittier with Plex. Instead of deleting everything and starting over, adding a new Jellyfin connection would be amazing to run in parallel with Plex.

**Comment by DerP4si** [comment]
absolutely! I know this comment doesnt add anything useful, but i wanted to show, there are many ppl that want this feature 😃

**Comment by MateusTorquato** [comment]
This would be a game changer for sure. Being able to run it with Jellyfin and Plex at the same time would be amazing

**Comment by clemone210** [comment]
I would like to be able to switch from Plex to Jellyfin without redoing everything, this feature request could come in handy!

**Comment by PixNyb** [comment]
I run both a plex and jellyfin server. My users have preferences for one of the two and i’d hate to be that person that forces them to either create a plex account or use their provided jellyfin account which they don’t use for jellyfin itself. Giving the user an option to log in with either, doesn’t necessarily have to be in the same form, would be an amazing addition.

My suggestion would be that logging in for the first time or importing users creates a separate user object (which i believe it already does) in the jellyseer db and the user can activate the other log in methods in their account if they want to use that?

Do correct me if i’m wrong however. If needed i might be able to take a look at implementing this as well

**Comment by baltermia** [comment]
Damn so people requested this 3 years ago and it's still not implemented? Bummer.

**Comment by Guruleenyc** [comment]
I also host Plex for many years and as of recently I also stood up a Jellyfin server as a secondary/backup. With all the changes on the Plex side of the house...I planning to move towards Jellyfin over the next few months as my primary media server.
At this time, it appears you can configure only one media server in Jellyseer or Overseer....
I'll manage both Overseer and Jellyseer for now, but hopefully support for multiple media servers is in the near future.

**Comment by butts136** [comment]
Hey, just want to know it this will be possible now with the new Seerr merge ?

Thanks

**Comment by fallenbagel** [comment]
> Hey, just want to know it this will be possible now with the new Seerr merge ?
> 
> Thanks

No. This issue will be closed as completed when it's implemented ;)

**Comment by amigthea** [comment]
I wrongly gave this for granted, reading that both are supported. But yeah, I agree that it would cover my case too, it's not so niche either I think? anyway awesome project, keep it up!

**Comment by Lumpish3007** [comment]
If you landed here from a search engine and are looking for the ability to swap between plex and Jellyfin check out [this discussion](https://github.com/seerr-team/seerr/discussions/2792).
